lib/v8.js

```diff
@@ -78,16 +78,24 @@ const { getOptionValue } = require('internal/options');
  * @param {string} [filename]
  * @param {{
  *   exposeInternals?: boolean,
- *   exposeNumericValues?: boolean
+ *   exposeNumericValues?: boolean,
+ *   path?: string,
+ *   encoding?: string
  * }} [options]
  * @returns {string}
  */
 function writeHeapSnapshot(filename, options) {
   if (filename !== undefined) {
     filename = getValidatedPath(filename);
   }
+
+  if (options?.path !== undefined) {
```

Comment:
Nit: I believe the general convention for options validation is to check for any nullish value, unless `null` has a discrete meaning for that option.

```suggestion
  if (options?.path != undefined) {
```

Reply:
Roger that

Reply:
Linter does not like that. :-)